src/components/molecules/VenuePartygoers/VenuePartygoers.tsx

```diff
@@ -16,8 +20,16 @@ const filterVenuePartygoers = (partygoers: User[], venue: Venue) => {
 export const VenuePartygoers = () => {
   const venue = useSelector(currentVenueSelector);
   const partygoers = useSelector(partygoersSelector) ?? [];
+  const parentVenue = useSelector(parentVenueSelector);
 
+  const currentVenueTitle = venue.headerTitle ?? "partygoers";
+  const headerTitle = parentVenue?.headerTitle ?? currentVenueTitle;
   const currentVenuePartygoers = filterVenuePartygoers(partygoers, venue);
```

Comment:
Probably wrap this with `useMemo` so we only have to re-filter the data when current venue or partygoers changes, rather than every render.

Reply:
[NB, Future Improvement]